renovate.json

```diff
@@ -7,5 +7,12 @@
   ],
   "constraints": {
     "pnpm": "8.6.11"
-  }
+  },
+  "packageRules": [
+    {
+      "matchPackageNames": ["@kong-ui-public/expressions", "@kong-ui-public/entities-routes"],
+      "groupName": "exclude monaco-editor from @kong-ui-public/expressions and @kong-ui-public/entities-routes",
+      "excludeDepNames": ["monaco-editor"]
+    }
+  ]
```

Comment:
issue: `matchPackageNames` is for the matched dependency package names, not the packages within the repository

I think what you actually want here is something like `matchFileNames` whereby you provide the path to the `package.json` of these packages, e.g. 

```json
{
  "packageRules": [
    {
      "description": "Pin monaco-editor dependency for the expressions package",
      "matchFileNames": ["packages/core/expressions/package.json"],
      "groupName": "expressions-monaco-editor",
      "matchPackageNames": ["monaco-editor"],
      "enabled": false
    }
  ]
}
```

Reply:
@adamdehaven Thanks. I've updated the configuration with the suggestion